server/src/ClientToDBAPI/ContactedPeople/mainRoute.ts

```diff
@@ -45,6 +46,37 @@ ContactedPeopleRoute.get('/familyRelationships', (request: Request, response: Re
     })
 });
 
+ContactedPeopleRoute.get('/contactStatuses', (request: Request, response: Response) => {
+    logger.info({
+        service: Service.SERVER,
+        severity: Severity.LOW,
+        workflow: 'Getting contact statuses',
+        step: 'launching graphql API request',
+        user: response.locals.user.id,
+        investigation: response.locals.epidemiologynumber
+    });
+    graphqlRequest(GET_ALL_CONTACT_STATUSES, response.locals).then((result: any) => {
+        logger.info({
```

Comment:
Sometimes graphql return error on result, so you should check if the result is returned as exppceted.
In addition, I would change the route to return the array and not all the result, if you do so you need to check on the client the header application/json